code/game/machinery/autolathe.dm

```diff
@@ -98,10 +98,13 @@
 	else if(!disabled)
 		ui_interact(user)
 
-/obj/machinery/autolathe/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = TRUE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state)
-	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+/obj/machinery/autolathe/ui_state(mob/user)
+	return GLOB.default_state
```

Comment:
is there any reason why the base machinery type doesn't just use GLOB.default_state by default? There's about 40-50 machine child types of `/obj/machinery` which override this proc only to all return the same thing. I've yet to come across an override that **doesn't** return the default state

Reply:
We specifically define these for each type that has a `ui_interact()` declaration. This is for clarity on what state is used for each type.